packages/e2e-tests/pages/patients/LabRequestPage/modals/IndividualLabRequestModal.ts

```diff
@@ -61,4 +70,50 @@ export class IndividualLabRequestModal extends LabRequestModalBase {
       await expect(testName).toBe(selectedTests[i]);
     }
   }
+
+  async validateRequestFinalisedPage({
+    requestingClinician,
+    requestedDateTime,
+    department,
+    priority,
+    expectedCategories,
+    expectedSampleDate,
+  }: ValidateRequestFinalisedPageParams) {
+    // Validate header values
+    await expect(this.requestingClinicianValue).toHaveText(requestingClinician || 'Unknown');
+    await expect(this.requestDateTimeValue).toHaveText(requestedDateTime);
+    await expect(this.departmentValue).toHaveText(department || 'Unknown');
+    await expect(this.priorityValue).toHaveText(priority || '-');
+
+    // Validate finalised table categories
+    const requestFinalisedCategoryItems = await this.getRequestFinalisedTableItems(expectedCategories.length, 'labTestCategory');
+    await expect(requestFinalisedCategoryItems).toEqual(expectedCategories);
+    // Validate finalised table sample dates
+    const requestFinalisedSampleDateItems = await this.getRequestFinalisedTableItems(expectedCategories.length, 'sampleDate');
+    let formattedSampleDate: string;
+    if (
+      expectedSampleDate &&
+      !isNaN(Date.parse(expectedSampleDate)) // valid date string
+    ) {
+      formattedSampleDate = format(new Date(expectedSampleDate), 'MM/dd/yyyy h:mm a');
+    } else {
+      formattedSampleDate = expectedSampleDate || 'Sample not collected';
+    }
+    for (let i = 0; i < expectedCategories.length; i++) {
+      await expect(requestFinalisedSampleDateItems[i]).toEqual(formattedSampleDate);
```

Comment:
```suggestion
      expect(requestFinalisedSampleDateItems[i]).toEqual(formattedSampleDate);
```
Not a biggie at all, but these `toEqual()` calls don't need an `await` as they don't return Promises :+1: